go/test/endtoend/cluster/cluster_util.go

```diff
@@ -292,3 +298,100 @@ func filterDoubleDashArgs(args []string, version int) (filtered []string) {
 
 	return filtered
 }
+
+// WriteDbCredentialToTmp writes json format db credentials to tmp directory
```

Comment:
(i know you're just copy-pasting from that now-deleted file, but —) can you punctuate these function comments? ❤️